main.go

```diff
@@ -59,6 +45,38 @@ func update() {
 	fmt.Println("Updated Successfully.")
 }
 
+func parseJSON(s string) {
+	//	Open the provided file
+	jsonFile, err := os.Open(s)
+	if err != nil {
+		fmt.Println("File Not Found")
```

Comment:
Return/Raise exception if we were unable to open the file for whatever reason. No point continuing with function processing